### PR TITLE
Hibernate crud invitation

### PR DESCRIPTION
**Pull request by KirylShalpuk**
included hibernate to CRUD for Invitation

### CODE REVIEW

File: src/main/java/app/dao/impl/InvitationDaoImpl.java [hunk 1]
```diff
@@ -2,175 +2,60 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
 
-import java.sql.*;
-import java.util.LinkedList;
 import java.util.List;
 
 @Repository
+@Transactional
```
Comment (AlexKolonitsky) [inline src/main/java/app/dao/impl/InvitationDaoImpl.java]: убрать

File: src/main/java/app/config/beans/HibernateConfig.java [hunk 2]
```diff
@@ -28,9 +28,9 @@
     @Value("${hibernate.dialect}")
     private String hibernateDialect;
     @Value("${hibernate.show_sql}")
-    private boolean showSql;
+    private String showSql;
```
Comment (AlexKolonitsky) [inline src/main/java/app/config/beans/HibernateConfig.java]: revert

**Review by AlexKolonitsky — CHANGES_REQUESTED** [review]

File: src/main/java/app/config/beans/HibernateConfig.java (hunk 2 above)
Comment (KirylShalpuk) [reply src/main/java/app/config/beans/HibernateConfig.java]: completed

File: src/main/java/app/dao/impl/InvitationDaoImpl.java (hunk 1 above)
Comment (KirylShalpuk) [reply src/main/java/app/dao/impl/InvitationDaoImpl.java]: completed

**Review by AlexKolonitsky — APPROVED** [review]

**Pull request merged**